spark/src/test/scala/org/apache/spark/sql/CometTestBase.scala

```diff
@@ -429,65 +429,131 @@ abstract class CometTestBase
     makeParquetFileAllTypes(path, dictionaryEnabled, 0, n)
   }
 
-  def makeParquetFileAllTypes(
-      path: Path,
-      dictionaryEnabled: Boolean,
-      begin: Int,
-      end: Int,
-      pageSize: Int = 128,
-      randomSize: Int = 0): Unit = {
-    val schemaStr =
+  def getAllTypesParquetSchema: String = {
```

Comment:
If we are renaming this method, I wonder if we should remove the `AllTypes` part since it does not generate all types. Perhaps `getPrimitiveTypesParquetSchema`?

Reply:
done